src/server/js/routes.js

```diff
@@ -0,0 +1,43 @@
+import express from 'express';
+
+import passport from './passport';
+import UserModel from './resources/users/model';
+
+const router = new express.Router();
+
+router.post('/login', (req, res) => {
```

Comment:
i think both of these routes should live in the 'user' model (as they're both actions on user entities)